noether_conversions/src/noether_conversions.cpp

```diff
@@ -134,7 +134,7 @@ visualization_msgs::Marker createMeshMarker(const std::string& mesh_file, const
   std::tie(m.scale.x , m.scale.y ,m.scale.z) = std::make_tuple(1.0,1.0,1.0);
   m.mesh_resource = "file://" + mesh_file;
   m.type = m.MESH_RESOURCE;
-  return std::move(m);
+  return m;
```

Comment:
I hate move() too.

Reply:
Pre c++11 it was necessary but now it prevents copy epsilon, so it should be avoided for function return values. I mainly use clang-tidy which will let you know when to add them or if you have added them where you should have not.